### DISCUSSION TITLE
Website Enhancement

### DISCUSSION THREAD

**Discussion by Kshitij978** [Ashutosh00710/github-readme-activity-graph#102] [General]
Reference to the pull request #100

**Comment by DalpatRathore** [comment]
My take on website enhancement
1. Responsive UI/UX - Mobile & Desktop
2. Enhanced text and background color contrast
3. Website logo
   -  I tinkered around the existing one - three proto types.
   <img width="250" src="https://user-images.githubusercontent.com/69510006/149984216-407c55a2-1c9e-460d-8237-38cfb162587b.png" />  <img width="250" src="https://user-images.githubusercontent.com/69510006/149985080-ba1b48f5-858f-49b2-bd7c-d01d83cb1795.png" />  <img width="250" src="https://user-images.githubusercontent.com/69510006/149985100-5e704f25-5d5f-4049-b40c-fceccc83c198.png" /> 
4. Gradient colors for **graph background** - will keep up with new trends and also give more room to theme contributors.

**Reply by Kshitij978** [reply]
Hey! @DalpatRathore I am thinking of going with the major changes first that includes: 
1. Improvement in overall UI/UX of the website (mainly the UX part) 
2. Full responsiveness.
3. Change in UI of the COPY button
4. A placeholder/skeleton which should be shown to the user at the place of the Graph to let them know their graph will be displayed there.
5. Loader feels a bit weird and it blocks the entire screen so we need to change that too.

I have made few of the above changes but haven't coded in JS yet. Below is the screenshot. Do suggest if it could be enhanced further.

(Don't mind the weird whitish area at the top. It's just a problem with the screenshot)

![WhatsApp Image 2022-01-17 at 9 54 50 PM](https://user-images.githubusercontent.com/42491256/150684134-69a9ea5f-6774-4a6b-a221-03f94655ddb4.jpeg)


I like the first modified logo by you. We can use that but let's work on the major changes first. 😃

P.S. I'm sorry for responding late. I have been sick from past few days. 🤧

**Reply by Ashutosh00710** [reply]
Great! @Kshitij978 @DalpatRathore 🚀, Let me know if you guys need me anywhere, I am here to help you ☺️

**Reply by DalpatRathore** [reply]
Hey! @Kshitij978 I hope you feel better now if not get well soon. I do agree with your proposed changes.

**Reply by DalpatRathore** [reply]
Hey! @Ashutosh00710, how about new feature - Gradient Colors for Graph Background
- ![Capture](https://user-images.githubusercontent.com/69510006/150689262-94058159-f414-4cb5-9d28-118dabecf33c.JPG)
- ![Capture1](https://user-images.githubusercontent.com/69510006/150689271-8f23ecf6-5bd0-4c93-bea4-ba512cde366f.JPG)

**Reply by Kshitij978** [reply]
Thank you, @Ashutosh00710 & @DalpatRathore  . If you guys think of any other major changes then please extend the list. 😄

**Reply by Ashutosh00710** [reply]
> Hey! @Ashutosh00710, how about new feature - Gradient Colors for Graph Background
> - ![Capture](https://user-images.githubusercontent.com/69510006/150689262-94058159-f414-4cb5-9d28-118dabecf33c.JPG)
> - ![Capture1](https://user-images.githubusercontent.com/69510006/150689271-8f23ecf6-5bd0-4c93-bea4-ba512cde366f.JPG)

@DalpatRathore , great idea go ahead for the implementation and let me know if you need any assistance 😊 I am here to help you.
